startups/models.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+
+class Startup(models.Model):
+    company_name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+    funding_stage = models.CharField(
+        max_length=100,
+        choices=[
+            ('idea', 'Idea'),
+            ('pre-seed', 'Pre-Seed'),
+            ('seed', 'Seed'),
+            ('series_a', 'Series A'),
+            ('series_b', 'Series B'),
+            ('growth', 'Growth'),
+        ],
+        default='idea'
+    )
+    location = models.CharField(max_length=255, blank=True)
+    industries = models.CharField(max_length=255, help_text="Comma-separated list of industries")
```

Comment:
Consider using a separate table or ManyToManyField for industries. Comma-separated fields can make querying and integrity checks harder.